sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Generated/Models/DataSourceCredentialEntity.cs

```diff
@@ -11,7 +11,7 @@
 namespace Azure.AI.MetricsAdvisor.Administration
 {
     /// <summary> The DataSourceCredential. </summary>
-    public partial class DataSourceCredentialEntity
+    public abstract partial class DataSourceCredentialEntity
```

Comment:
While I agree this should have been made `abstract` during development, this class has already been shipped as part of the Metrics Advisor public API, so we can't change it without architect review. Could we somehow skip these changes for Metrics Advisor?

The same reasoning is true for the `MetricFeedback` and the `NotificationHook` classes.

Reply:
Reverted this changes for Metrics Advisor.